prow_config.yaml

```diff
@@ -26,6 +26,24 @@ workflows:
     params:
       installIstio: true
       workflowName: deployapp-istio
+  # kfctl test runs tests on gke using the new kfctl go binary
+  - app_dir: kubeflow/kubeflow/testing/workflows
+    component: kfctl_go_test
+    name: kfctl-go
+    job_types:
+      - presubmit
+      - postsubmit
+      - periodic
+    include_dirs:
+      - bootstrap/*
+      - deployment/*
+      - kubeflow/*
+      - scripts/*
```

Comment:
Can we add a TODO for removing scripts/* in future?

Reply:
I went ahead and removed it; I don't think it should depend on scripts since its just go code.